p11-kit/meson.build

```
@@ -0,0 +1,290 @@
+install_headers('deprecated.h',
+                'iter.h',
+                'p11-kit.h',
+                'pin.h',
+                'remote.h',
+                'uri.h',
+                subdir: 'p11-kit-1/p11-kit')
+
+libp11_kit_internal_sources = [
+  'conf.c',
+  'log.c',
+  'filter.c',
+  'rpc-transport.c',
+  'rpc-message.c',
+  'rpc-client.c'
+]
+
+virtual_fixed_generated_h = custom_target('generate virtual-fixed-generated.h',
```

You can add directly the result of the `custom_target` to the sources so you avoid creating an extra variable:

```suggestion
libp11_kit_internal_sources += custom_target('generate virtual-fixed-generated.h',
```